lib/rubycritic/generators/console_report.rb

```diff
@@ -0,0 +1,52 @@
+require "colorize"
+
+module Rubycritic
+  module Generator
+    class ConsoleReport
+      def initialize(analysed_modules)
+        @analysed_modules = analysed_modules
+      end
+
+      def generate_report
+        reports = @analysed_modules.sort.map do |analysed_module|
+          generate_module_report analysed_module
+        end
+        puts reports.join("\n")
+      end
+
+      private
+
+      def generate_module_report(mod)
+        report = <<-EOF
+#{mod.name.send(mod.rating.color)}:
+    Rating:       #{mod.rating.to_s.send(mod.rating.color)}
```

Comment:
Same as above. Apart from that it's hard to read and hard to maintain. We should refactor our approach of how to access those attributes using a separate report object so that we can use proper composition and delegation.